src/test/java/com/google/api/generator/gapic/composer/grpcrest/GrpcRestTestProtoLoader.java

```diff
@@ -82,4 +83,35 @@ public GapicContext parseShowcaseEcho() {
         .setTransport(getTransport())
         .build();
   }
+
+  public GapicContext parseShowcaseWicked() {
+    FileDescriptor fileDescriptor = WickedOuterClass.getDescriptor();
+    ServiceDescriptor messagingService = fileDescriptor.getServices().get(0);
+    assertEquals("Wicked", messagingService.getName());
+
+    Map<String, Message> messageTypes = Parser.parseMessages(fileDescriptor);
```

Comment:
This logic looks duplicated with the above method's. If this pattern will be repeated for further clients, consider refactoring this common logic out into something like `createGapicContext(fileDescriptor)`